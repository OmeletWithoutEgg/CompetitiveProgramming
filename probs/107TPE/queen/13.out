-1162507.19 -42007396.46
