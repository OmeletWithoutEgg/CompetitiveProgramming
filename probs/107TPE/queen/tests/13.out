-12948518.52 15047900.86
